main.py

```diff
@@ -145,6 +145,8 @@ def compress_tar(self, path, target):
         files = os.listdir(path)
         tar = tarfile.open(target, 'w')
         for file in files:
+            if file == ".DS_Store":
```

Comment:
I think its generally a good idea to keep a tight scope for PR's. For this one though I think it's fine to exclude all hidden files since I couldn't find any of them in the original FW.

I get similar `.directory` files created by Dolphin file manager, and there's probably other such cases too. So let's go for excluding all of them. Thanks for the PR! :+1: